wagtailvideos/static/wagtailvideos/js/add-multiple.js

```diff
@@ -136,6 +136,39 @@ $(function() {
         });
     });
 
+    $('#locations_button').click(function(e) {
```

Comment:
button should be called something different - add_storage? add_direct?

Reply:
`add_storage` seems good for me.